README.md

```diff
@@ -229,15 +229,9 @@ The trigger takes the format:
 - **/test metal3-e2e-1-28-1-29-upgrade-test-main**
 - **/test metal3-e2e-1-27-1-28-upgrade-test-main**
 - **/test metal3-e2e-1-26-1-27-upgrade-test-main**
+- **/test metal3-e2e-1-28-1-29-upgrade-test-release-1-7**
 - **/test metal3-e2e-1-28-1-29-upgrade-test-release-1-6**
-- **/test metal3-e2e-1-27-1-28-upgrade-test-release-1-6**
-- **/test metal3-e2e-1-26-1-27-upgrade-test-release-1-6**
-- **/test metal3-e2e-1-28-1-29-upgrade-test-release-1-5**
-- **/test metal3-e2e-1-27-1-28-upgrade-test-release-1-5**
 - **/test metal3-e2e-1-26-1-27-upgrade-test-release-1-5**
-- **/test metal3-e2e-1-28-1-29-upgrade-test-release-1-4**
-- **/test metal3-e2e-1-27-1-28-upgrade-test-release-1-4**
-- **/test metal3-e2e-1-26-1-27-upgrade-test-release-1-4**
 
```

Comment:
Are these tests removed from the jjbs?

Reply:
This were never added to jjb. We have one k8s upgrade test per release